src/sql-parser/src/keywords.txt

```diff
@@ -33,6 +33,7 @@ Array
 As
 Asc
 At
+# Authority
```

Comment:
I dislike leaving these commented out like this--I think they could just get removed, or if that's anathema to you, grouped together with a `TODO` placed atop it